### ISSUE TITLE
SQL: table constraint syntax isn't supported

### ISSUE THREAD

**Issue by tdakkota** [genjidb/genji#220]
Table constraint is constraint declaration like
```SQL
CREATE TABLE foo(
    ID integer,
    PRIMARY KEY(ID)
);
```
Composite key can be defined like this
```SQL
CREATE TABLE foo(
    First integer,
    Second integer,
    PRIMARY KEY(First, Second)
);
```
It's part of SQL-92, it would be great to support this form.

**Comment by asdine** [comment]
Yes definitely, composite keys are scheduled for [v0.9.0](https://github.com/genjidb/genji/milestone/10), so this is related to #35

**Comment by asdine** [comment]
Fixed by #382
